code/game/objects/items/weapons/storage/backpack.dm

```diff
@@ -212,11 +212,33 @@ obj/item/weapon/storage/backpack/ww2/american
 		new/obj/item/ammo_casing/rocket/pg7v(src)
 		new/obj/item/ammo_casing/rocket/pg7v(src)
 		new/obj/item/ammo_casing/rocket/pg7v(src)
+
+/obj/item/weapon/storage/backpack/heavyrucksack
+	name = "heavy rucksack"
+	desc = "A big heavyduty rucksack made for big, heavy objects."
+	icon_state = "heavyrucksack"
+	item_state = "backpack"
+	base_icon = "heavyrucksack"
+	w_class = 5
+	slot_flags = SLOT_BACK
+	max_w_class = 5
+	max_storage_space = 28
+
+/obj/item/weapon/storage/backpack/heavyrucksack/atgm
+	New()
+		..()
+		new/obj/item/ammo_casing/rocket/atgm(src)
+		new/obj/item/ammo_casing/rocket/atgm(src)
+		new/obj/item/ammo_casing/rocket/atgm/apcr(src)
+		new/obj/item/ammo_casing/rocket/atgm/apcr(src)
+		new/obj/item/ammo_casing/rocket/atgm/he(src)
+		new/obj/item/ammo_casing/rocket/atgm/he(src)
+
 /obj/item/weapon/storage/backpack/civbag
 	name = "backpack"
 	desc = "A big backpack made for long walks."
 	icon_state = "civback"
-	item_state = "civback"
+	item_state = "backpack"
```

Comment:
Will need to eventually add in proper in-hands, if that's the reason you've changed it.